pkg/cloud/gcp/provider.go

```diff
@@ -877,69 +879,73 @@ func (gcp *GCP) parsePage(r io.Reader, inputKeys map[string]models.Key, pvKeys m
 								continue
 							} else if strings.Contains(strings.ToUpper(product.Description), "RAM") {
 								if instanceType == "custom" {
-									log.Debug("RAM custom sku is: " + product.Name)
+									log.Debugf("GCP Billing API: RAM custom sku '%s'", product.Name)
 								}
 								if _, ok := gcpPricingList[candidateKey]; ok {
+									log.Debugf("GCP Billing API: key '%s': RAM price: %f", candidateKey, hourlyPrice)
```

Comment:
Unified these logs so that they're more legible and clearly related to each other:
```
{"level":"debug","time":"2023-08-11T16:46:04-06:00","message":"GCP Billing API: key 'us-central1,a2highgpu,ondemand': RAM price: 0.004237"}
{"level":"debug","time":"2023-08-11T16:46:04-06:00","message":"GCP Billing API: key 'us-central1,a2highgpu,ondemand,gpu': RAM price: 0.004237"}
{"level":"debug","time":"2023-08-11T16:46:04-06:00","message":"GCP Billing API: key 'us-central1,a2highgpu,ondemand': CPU price: 0.031611"}
{"level":"debug","time":"2023-08-11T16:46:04-06:00","message":"GCP Billing API: key 'us-central1,a2highgpu,ondemand,gpu': CPU price: 0.031611"}
{"level":"debug","time":"2023-08-11T16:46:04-06:00","message":"GCP Billing API: key 'asia-southeast1,t2dstandard,ondemand': RAM price: 68.205000"}
{"level":"debug","time":"2023-08-11T16:46:04-06:00","message":"GCP Billing API: key 'asia-southeast1,t2dstandard,ondemand,gpu': RAM price: 68.205000"}
{"level":"debug","time":"2023-08-11T16:46:04-06:00","message":"GCP Billing API: key 'asia-southeast1,t2dstandard,ondemand': CPU price: 508.934997"}
{"level":"debug","time":"2023-08-11T16:46:04-06:00","message":"GCP Billing API: key 'asia-southeast1,t2dstandard,ondemand,gpu': CPU price: 508.934997"}
{"level":"debug","time":"2023-08-11T16:46:04-06:00","message":"GCP Billing API: key 'us-central1,e2medium,ondemand': CPU price: 327.173848"}
{"level":"debug","time":"2023-08-11T16:46:04-06:00","message":"GCP Billing API: key 'us-central1,e2medium,ondemand,gpu': CPU price: 327.173848"}
{"level":"debug","time":"2023-08-11T16:46:04-06:00","message":"GCP Billing API: key 'us-central1,e2standard,ondemand': CPU price: 327.173848"}
{"level":"debug","time":"2023-08-11T16:46:04-06:00","message":"GCP Billing API: key 'us-central1,e2standard,ondemand,gpu': CPU price: 327.173848"}
{"level":"debug","time":"2023-08-11T16:46:04-06:00","message":"GCP Billing API: key 'us-central1,e2medium,ondemand': RAM price: 43.852950"}
{"level":"debug","time":"2023-08-11T16:46:04-06:00","message":"GCP Billing API: key 'us-central1,e2medium,ondemand,gpu': RAM price: 43.852950"}
{"level":"debug","time":"2023-08-11T16:46:04-06:00","message":"GCP Billing API: key 'us-central1,e2standard,ondemand': RAM price: 43.852950"}
{"level":"debug","time":"2023-08-11T16:46:04-06:00","message":"GCP Billing API: key 'us-central1,e2standard,ondemand,gpu': RAM price: 43.852950"}
```